scripts/mwa_metadb_utils.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+import logging
+import argparse
+
+from vcstools.metadb_utils import getmeta, write_obs_info, get_obs_array_phase, obs_max_min, calc_ta_fwhm, get_best_cal_obs
```

Comment:
Do you think it's worthwhile changing the name of this file now? since it isn't really acting as a bunch of utilities and more like a script

Reply:
I did think about it but then everyone would have to change their habits

Reply:
Yea that's a consideration too. Up to you